tor/strings/en_US.yml

```diff
@@ -201,29 +201,29 @@ formatting_issues:
 
 
     \    <header>\n
-    \   \n
-    \   ---\n
-    \   \n
-    \   <content>\n
-    \   \n
-    \   ---\n
-    \   \n
-    \   <footer>"
+    \    \n
```

Comment:
I removed a space from these because they were displaying as one space too far without them -- it's a weird quirk that I don't understand the meaning behind. Same with the extra space behind line 217.